src/components/Error.tsx

```diff
@@ -11,20 +12,25 @@ export interface ErrorPropTypes {
 }
 
 const EventId = styled.div`
-  font-size: 0.9rem;
+  font-size: 1.4rem;
   color: ${colors.palette.neutralMedium};
-  margin-top: 1.2rem;
+  margin-top: 1.6rem;
 `;
 
 export const Error = ({ heading, children, ...props }: ErrorPropTypes) => {
   const context = React.useContext(ErrorContext);
+  const [lastEventId, setLastEventId] = React.useState<string | undefined>();
+
+  React.useEffect(() => {
+    setLastEventId(Sentry.lastEventId());
+  }, [Sentry.lastEventId()]);
```

Comment:
This isn't a prop so I don't think it's going to work the way you want it to. Pretty sure this is the same as just calling lastEventId in your render

Reply:
Oops good catch, I wasn't thinking about it... I guess the only way to do this is with an interval: https://github.com/openstax/ui-components/pull/18/commits/09cc0202053def38c830c7387c594a4163455455#diff-43c27d73cea3bdeb585e9215204919e1b82cc0ae3edf657dbb7b2d44bbe110c8R29